packages/aragon-wrapper/src/cache/index.js

```diff
@@ -40,14 +40,15 @@ export default class Cache {
     return `${this.prefix}.${key}`
   }
 
-  set (key, value) {
+  async set (key, value) {
     // Some lowdb adapters are synchronous while others are asynchronous so
     // let's always wrap it in a promise
-    return Promise.resolve(this.db.set(
+    await Promise.resolve(this.db.set(
       this.getCacheKeyPath(key),
       value
     ).write())
-      .then(() => this.changes.next({ key, value }))
+     
+    this.changes.next({ key, value })
```

Comment:
@0x6431346e I don't think this has broken anything, but I think this has changed the API of `set()`. We should return the value here to keep it the same as before.

Reply:
Seems to return `undefined` though, the docs also say `public next(value: T): void`  :relieved: 
http://reactivex.io/rxjs/class/es6/MiscJSDoc.js~ObserverDoc.html#instance-method-next

Reply:
Oh good catch :). I just assumed it would return the observable.